CONVENTIONS.md

```diff
@@ -0,0 +1,78 @@
+# Agreed Conventions
+
+## Code structure
+
+To enable ease of understanding and consistency, we recommend following the structure as such:
+
+> Note: Add headers to the respective sections
+
+```tsx
+// component-file.tsx
+
+// Import statements
```

Comment:
is there a preference for imports to be `"../color/color"` vs `"../color"` vs `"src/color"`? (other than cases where importing from the index may result in some circular dep)

Reply:
it can't be `src/color` due to circular dep concerns. I'll add that in.